tests/Unit/Functions/Options/TestExcludeDeferJS.php

```diff
@@ -1,7 +1,7 @@
 <?php
 namespace WP_Rocket\Tests\Unit\Functions\Options;
 
-use PHPUnit\Framework\TestCase;
+use WP_Rocket\Tests\Unit\TestCase;
 use Brain\Monkey;
```

Comment:
You can remove this line too here, and in the `setUp()` function on line 12, remove the `Monkey\setUp();` while keeping the `parent::setUp()`